spec/system/candidate_interface/course_selection/candidate_arrives_from_find_with_the_maximum_amount_of_unsuccessful_choices_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate arrives from Find with provider and course params', :continuous_applications do
+  include CandidateHelper
+
+  scenario 'The candidate has hit the maximum amount of unsuccessful choices per cycle' do
+    given_there_is_a_provider_with_a_course_that_is_only_accepting_applications_on_apply
+
+    when_i_follow_a_link_from_find
+    then_i_am_redirected_to_the_create_account_or_sign_in_path
+
+    given_i_am_signed_in
```

Comment:
Can we rename this step to given_i_reach_the_max_limit_of_unsuccessful_applications (or similar)?